lambda/src/createPetFunction.js

```diff
@@ -54,7 +54,7 @@ const uploadImageToS3 = async (key, buffer) => {
 const insertIntoDynamo = async (id, type, imageFilename, headers) => {
   const db = new DynamoDB.DocumentClient();
 
-  let client = null;
+  let client = "Unkown";
```

Comment:
Didn't you get highest marks on English in high school?

Reply:
ESL